perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/build.sh

```diff
@@ -41,6 +41,24 @@ CONFIG_NAME=$(echo "$config" | jq -r '.config_name')
 GCSFUSE_FLAGS=$(echo "$config" | jq -r '.gcsfuse_flags')
 BRANCH=$(echo "$config" | jq -r '.branch')
 END_DATE=$(echo "$config" | jq -r '.end_date')
+# Get the value of the config_file_flags_as_json key
+CONFIG_FILE_FLAG_JSON=$(jq -r '.["config_file_flags_as_json"]' <<< $config )
+
+# Create config_flags.yml file from json.
+CONFIG_FILE_YML="${KOKORO_ARTIFACTS_DIR}/config_flags.yml"
+if [ "$CONFIG_FILE_FLAG_JSON" != null ];
+then
+  # Create JSON file to capture value of $CONFIG_FILE_FLAG_JSON
+  CONFIG_FILE_JSON="${KOKORO_ARTIFACTS_DIR}/config_flags.json"
+  echo "$CONFIG_FILE_FLAG_JSON" >> $CONFIG_FILE_JSON
+
+  jq -c -M . $CONFIG_FILE_JSON > $CONFIG_FILE_YML
+  GCSFUSE_FLAGS="$GCSFUSE_FLAGS --config-file $CONFIG_FILE_YML "
+
+  rm $CONFIG_FILE_JSON
+fi
+# Create string of config file value for fetching data from big query table.
```

Comment:
nit: Create string of config file (value -> content)

Reply:
Also, what does it do? Does it compress the json content?

Reply:
Tt uses the jq command to convert the value of the CONFIG_FILE_FLAG_JSON variable to a JSON string and then stores the result in the CONFIG_FILE_STRING variable.